polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -1221,36 +1184,6 @@ function IndexTableBase({
     handleSelectionChange(SelectionType.Page, checked);
   }
 
-  function renderStickyHeading(heading: IndexTableHeading, index: number) {
```

Comment:
Function has been merged with `renderHeading` above since they were _meant_ to be identical but had diverged in ways that introduced bugs in the sticky heading. By combining them, it ensures no divergence and consistent UX.